www/components/Attendies.js

```diff
@@ -0,0 +1,26 @@
+import styles from "../styles/Attendies.module.css";
+import Chip from "./Chip";
+import Input from "./Input";
+function Attendies() {
+  return (
+    <div className={styles.attbox}>
+      <div className={styles.attW}>
+        <h2>Attendies</h2>
+        Owner
+        <Chip text="Tom Hudson" icon={true}/>
+        <br />
+        Participant(s)
+        <div className={styles.atts}>
```

Comment:
It is generally a good idea to have all text within its own `<p>` or `<h>` tag so that if there are global styles for those tags they take effect. Can you wrap those titles in something? Also, people generally don't like to see `<br>` tags as they are inconsistend between browsers.